index.html

```diff
@@ -229,5 +229,32 @@ <h2 class="categorie-title">
                 </li>
             </ul>
         </section>
+        <section class="form" id="form-id">
+            <h1 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <form action="https://formspree.io/f/xvongony" method="POST" class="form-input">
+                <label class="input-name">
+                    <input type="text" class="full-name"name="full-name" id="full-name" maxlength="30" placeholder="Full name" required>
+                </label>
+                <label class="input-email">
+                    <input type="email" class="email" name="email" id="email" placeholder="Email address" required>
+                </label>
+                <label class="input-text">
+                    <textarea name="text_area" id="text_area" maxlength="500" placeholder="Enter text here" required></textarea>
+                </label>
+                <button type="submit" class="submit-button">Get in touch</button>
+            </form>
+        </section>
+        <footer>
+            <ul class="contact">
+                <li class="icon"><a href=""><img src="assets/contact/twitter-icon.svg" alt="twitter icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/linkedin-icon.svg" alt="linkedin icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/medium-icon.svg" alt="medium icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/github-icon.svg" alt="github icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/angellist-icon.svg" alt="angellist icon"></a></li>
+            </ul>
```

Comment:
-[OPTIONAL] Kindly consider adding your social media links in the `href=""` attribute of `a` tags. This will allow visitors to your portfolio website to look at your professional work.